core/src/main/java/org/apache/druid/timeline/partition/DimensionRangeShardSpec.java

```diff
@@ -279,25 +253,14 @@ public <T> PartitionChunk<T> createChunk(T obj)
     }
   }
 
-  private boolean isInChunk(InputRow inputRow)
-  {
-    return isInChunk(dimensions, start, end, inputRow);
-  }
-
   public static boolean isInChunk(
```

Comment:
You could probably remove this method. I don't think it is needed anymore.

Reply:
done